src/components/Book.js

```diff
@@ -13,22 +13,37 @@ export default function Book({ book }) {
 
   return (
     <div className="bookCard">
-      <div className="">
+      <div className="bookDetails">
         <p className="genre">{book.category}</p>
         <p className="title">{book.title}</p>
         <p className="author">{book.author}</p>
-        <ul>
```

Comment:
Please ensure that the styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)tyling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/).

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/163595979-8c017c49-9c42-43e8-a6e2-7e08b1fa9b0e.png) | ![image](https://user-images.githubusercontent.com/10332499/163595942-a25de58f-247e-46d7-9f10-4ef404f41771.png)

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/163596117-a763ad19-0c46-4bbd-a9a6-3cd115098de2.png) | ![image](https://user-images.githubusercontent.com/10332499/163596090-f28d4180-c25d-4dee-89e0-856f7886736f.png)


Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/163596278-038b3df8-4d27-44f7-9a50-712d835f18e1.png) | ![image](https://user-images.githubusercontent.com/10332499/163596253-162826e6-c9fd-4823-bce6-c81f2638216b.png)